modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/DistributionZoneManagerConfigurationChangesTest.java

```diff
@@ -332,6 +183,8 @@ void testSeveralZoneCreationsUpdatesTriggerKey() throws Exception {
     void testDataNodesNotPropagatedAfterZoneCreation() throws Exception {
         keyValueStorage.put(zonesChangeTriggerKey(1).bytes(), longToBytes(100));
 
+        clearInvocations(keyValueStorage);
```

Comment:
Why is this call needed?

Reply:
DistributedZoneManager can write to metastorage during initialization.
This just reset Mockito counters in Spy object.

Reply:
I think it's better to improve the verification method instead. Currently we use `verify(keyValueStorage, timeout(1000).times(1)).invoke(any())`, maybe we can verify actual invocation arguments instead

Reply:
It doesn't seems possible, argument is a general class. `StatementResult invoke(If iif)`

Reply:
that's a pity =( Ok then